ramls/subjecttype.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A subject type",
```

Comment:
I suggest adding some `code` field that will be unique, non-changeable, and helps identify each type in case of name change.

Reply:
But what will its value be for subject types with the source folio? abbreviation?

Reply:
Hey @psmagin ! we have discussed about that in our BE sync, and decided not to include this field in subject type as subject type should be identical as identifier type, and code will be included in subject itself. cc: @KaterynaSenchenko, @RuslanLavrov